packages/api/portfolioDrafts/createPortfolioDraft.ts

```diff
@@ -44,3 +46,8 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
     return DATABASE_ERROR;
   }
 }
+
+// IP logging middy
+const middyHandler = middy(handler);
+middyHandler.use(IpCheckerMiddleware());
+export { middyHandler };
```

Comment:
I would suggestion staying consistent with the other files and change `handler` to  `baseHandler` and use `handler` for the `baseHandler` wrapped with middy (example below). From my understanding `handler` is the default method invoked by lambdas ([AWS Lambda Functions - handler](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html)). There were other ones that were similar to this, so you would want to update them too.  
```ts
export const handler = middy(baseHandler)
    .use()
```